bindings/rust/wasmedge-sys/src/instance/memory.rs

```diff
@@ -1,14 +1,42 @@
+//! Defines WasmEdge Memory and MemType structs.
+//!
+//! A WasmEdge `Memory` defines a linear memory as described by `MemType`.
+//! `MemType` specifies the limits on the size of a memory by a range. The start of
+//! the limit range specifies min size (initial size) of that memory, while the end
+//! restricts the size to which the memory can grow later.
+
 use crate::{error::check, wasmedge, Error, WasmEdgeResult};
-use std::ops::Range;
+use std::ops::RangeInclusive;
 
+/// Struct of WasmEdge Memory.
+///
+/// A WasmEdge [`Memory`] defines a linear memory as described by [`MemType`].
 #[derive(Debug)]
 pub struct Memory {
     pub(crate) ctx: *mut wasmedge::WasmEdge_MemoryInstanceContext,
     pub(crate) registered: bool,
 }
 impl Memory {
-    /// Create a Memory instance
-    pub fn create(limit: Range<u32>) -> WasmEdgeResult<Self> {
+    /// Create a new [`Memory`] to be associated with the given capacity limit.
+    ///
+    /// # Arguments
+    ///
+    /// - `limit` specifies a inclusive range of the page count in the [`Memory`]. For example,
+    /// (10..=20) means the lower bound (inclusive) of the page count is 10 ,while the upper bound (inclusive) 20.
+    ///
+    /// # Errors
+    ///
+    /// If fail to create a [`Memory`], then an error is returned.
+    ///
+    /// # Example
+    ///
+    /// ```ignore
```

Comment:
Why does this example ignore?

Reply:
The "ignore" here notifies cargo not to do doc test against this piece of code. Thanks!